packages/shared/styles/components/SfList.scss

```diff
@@ -18,4 +18,16 @@
     font-size: $font-size-big-mobile;
     font-family: $body-font-family-secondary;
   }
+  &__item-padding {
```

Comment:
It doesn't follow BEM convention. Modifiers are prefixed with `--`. Also padding should be the same for every `SfList` element according to designs. Is it too small for this use case? Check line-height/height proportion with designs please